src/Symfony/Component/HttpKernel/Controller/TraceableControllerResolver.php

```diff
@@ -49,18 +49,4 @@ public function getController(Request $request)
 
         return $ret;
     }
-
-    /**
-     * {@inheritdoc}
-     */
-    public function getArguments(Request $request, $controller)
```

Comment:
Internally but someone might depend on this. I would follow @fabpot and not change the interface as we have better solutions (more complex but working).